cartography/intel/aws/ec2.py

```diff
@@ -1031,6 +1071,50 @@ def load_ec2_vpcs(neo4j_session, data, region, current_aws_account_id, aws_updat
         )
 
 
+TAG_RESOURCE_TYPE_MAPPINGS = {
+    'instance': {'label': 'EC2Instance', 'property': 'instanceid'},
+    'network-interface': {'label': 'NetworkInterface', 'property': 'id'},
+    'security-group': {'label': 'EC2SecurityGroup', 'property': 'id'},
+    'subnet': {'label': 'EC2Subnet', 'property': 'subnetid'},
+    'vpc': {'label': 'AWSVPC', 'property': 'id'},
+}
+
+TAG_INGEST_TEMPLATE = Template("""
+MATCH (n:$label{$property: {id}})
+WITH n
+MERGE (tag:Tag{key: {Key}})
+ON CREATE SET tag.firstseen = timestamp()
+WITH tag, n
+MERGE (n)<-[r:TAGGED]-(tag)
```

Comment:
For consistency, would it make sense to reverse the directionality here?

At least in the case of GCP, this way makes more sense to me: 
```
(GCPInstance)-[:TAGGED]->(GCPNetworkTag)
```
https://github.com/lyft/cartography/blob/master/docs/schema/gcp.md#relationships-5